src/views/RecIM/components/Forms/MatchForm/index.jsx

```diff
@@ -132,15 +127,15 @@ const MatchForm = ({ closeWithSnackbar, openMatchForm, setOpenMatchForm, activit
         teamIDs.push(match.Activity.Team.find((_team) => team.ID === _team.ID).Name),
       );
       return {
-        StartTime: match.Time,
+        StartTime: match.StartTime,
         StatusID:
           matchStatus.find((type) => type.Description === match.Status) == null
             ? ''
             : matchStatus.find((type) => type.Description === match.Status).Description,
         SurfaceID:
-          surfaces.find((type) => type.Description === match.Surface) == null
+          surfaces.find((type) => type.Name === match.Surface) == null
             ? ''
-            : surfaces.find((type) => type.Description === match.Surface).Description,
+            : surfaces.find((type) => type.Name === match.Surface).Name,
```

Comment:
As with activityTypes above, can't you use `match.Surface` directly now?:
```suggestion
          match.Surface,
```